### PR TITLE
Added page about maps and updated nav

### PR DESCRIPTION
**Pull request by victoriaspek**
Saw questions around how to upload custom maps a few times when looking through Discourse and realized our docs don't directly answer that question right now, so I added a page to the admin guide.

### CODE REVIEW

File: docs/administration-guide/20-custom-maps.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Custom Maps
+
+By default, Metabase uses OpenStreetMaps for map visualizations, but there are a few customization options.
+
+### Finding Map Settings
+
+To find the map settings:
+
+1. Click the gear icon in the upper right.
```
Comment (flamber) [inline docs/administration-guide/20-custom-maps.md]: 1. Click the gear icon in the upper right, and then select **Admin**

Comment (mbvictoria) [reply docs/administration-guide/20-custom-maps.md]: Thanks, fixed!

File: docs/administration-guide/20-custom-maps.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+2. Select `Maps` from the navigation on the left.
+
+![Map Settings](images/MapSettings.png)
+
+### Map Tile Server
+
+If you'd like to use a map tile server other than OpenStreetMaps, simply add the map tile server URL to the first field on the page.
```
Comment (mazameli) [inline docs/administration-guide/20-custom-maps.md]: Should we quickly explain why you would want to use a different map tile server?

File: docs/administration-guide/20-custom-maps.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+### Custom Maps
+
+Metabase also provides the option to add custom regional maps via GeoJSON. To get started, the `Add a map` button. A modal will appear. You will need to enter the name you'd like to use for the map, the URL for the GeoJSON file and the properties that you'd like to use to specify your region's identifier and display name.
```
Comment (mazameli) [inline docs/administration-guide/20-custom-maps.md]: "To get started, the `Add a map` button" is missing the word "click."

"To get started, click the `Add a map` button"

Comment (mazameli) [inline docs/administration-guide/20-custom-maps.md]: Feels like we might want to give an example here. Like, I'm not 100% sure what filling out "the properties that you'd like to use to specify your region's identifier and display name" entails.

File: docs/administration-guide/20-custom-maps.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+1. Click the gear icon in the upper right.
+2. Select `Maps` from the navigation on the left.
+
+![Map Settings](images/MapSettings.png)
+
+### Map Tile Server
+
+If you'd like to use a map tile server other than OpenStreetMaps, simply add the map tile server URL to the first field on the page.
+
+### Custom Maps
```
Comment (mazameli) [inline docs/administration-guide/20-custom-maps.md]: I think we could distinguish region maps from world/pin maps a bit more explicitly. This is also the second "Custom Maps" heading on this page, so I'd suggest changing this to "Custom Region Maps".

File: docs/administration-guide/20-custom-maps.md (hunk 3 above)
Comment (mazameli) [inline docs/administration-guide/20-custom-maps.md]: Do we have any good suggestions for resources/links either to find good GeoJSON region maps, or how to create them?

**Review by mazameli — CHANGES_REQUESTED** [review]

File: docs/administration-guide/20-custom-maps.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+If you need a map that focuses on a specific region, you may want to upload a custom GeoJSON map. 
+
+To get started, click the `Add a map` button. A modal will appear, asking for:
+
+ * The name of the map
+ * The URL of the GeoJSON file
+ * The JSON property that Metabase should use as your region's identifier (a distinct way of identifying this specific region map) 
+ * The JSON property that Metabase should use as the sdisplay name (how the name of the region map appears in the user interface)
+
+If you do not currently have a GeoJSON file, there are many tools available to create one, such as [GeoJSON.io](geojson.io/). If you're just interested in reading more about GeoJSON, we recommend that you start [here](https://geojson.org/).
```
Comment (flamber) [inline docs/administration-guide/20-custom-maps.md]: geojson.io is http-only sadly, so the URL should be http://geojson.io/
I would also recommend Mapshaper - https://mapshaper.org/

Comment (mbvictoria) [reply docs/administration-guide/20-custom-maps.md]: Thanks!!

**Review by mazameli — APPROVED** [review]

**Pull request merged**